packages/maps/src/Client.ts

```diff
@@ -114,6 +114,16 @@ export interface AggregationColumn {
   aggregated_column: string;
 }
 
+export interface StatsColumn {
+  topCategories: number;
+  includeNulls: boolean;
+}
+
+export interface Sample {
+  num_rows: number;
```

Comment:
I commented this in other PR, but I think we must be consistent to avoid errors. If this is a public interface, IMO we should use `numRows` and `includeColumns` and then translate them to what Maps API requires internally

Reply:
Make sense, but maybe we can create a separate story for that, we also need to do the same for DO.